pages/04.HDM Doc 2.2/15.release-notes/default.md

```diff
@@ -11,6 +11,7 @@ What is new in HDM 2.2.1 Release?
 *   Provides support for Cold, TBC and Warm Migration to VMware Cloud Director.
 *   Highly efficient, proprietary, data transport for Cold, TBC and Warm migration.
 *   WAN disconnects lasting more than 2 minutes will be recovered.
+*   Static IP support is provided during migration on VC-VCD.
```

Comment:
Doesn't seem like we have mentioned the short forms VC & VCD in this section before this.

Also keeping in line with the phrases used in this section earlier better to use "Static IP support is provided during migration to VMware Cloud Director."

Reply:
Agreed. Manpreet, please update this.

Reply:
Resolved..